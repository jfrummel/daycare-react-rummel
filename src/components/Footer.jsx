import React from 'react';

const Footer = () => (
    <footer>
        <p>&copy; JR Designs 2020</p>
    </footer>
);

export default Footer;