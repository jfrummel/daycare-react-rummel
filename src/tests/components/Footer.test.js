import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

test("renders footer", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
});