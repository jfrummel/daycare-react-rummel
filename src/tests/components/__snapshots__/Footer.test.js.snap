// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer 1`] = `
<footer>
  <p>
    Â© JR Designs 2020
  </p>
</footer>
`;
